src/Directive/VersionAddedDirective.php

```diff
@@ -23,12 +23,15 @@ public function getName(): string
     public function processSub(Parser $parser, ?Node $document, string $variable, string $data, array $options): ?Node
     {
         $wrapperDiv = $parser->renderTemplate(
-            'directives/version-added.html.twig',
+            'directives/admonition.html.twig',
```

Comment:
The `version-added.html.twig` file can be deleted now, right?

Reply:
Or is the old template still used somehow?

Reply:
Yes, sorry! I forgot about that 😞   Please, delete it if you can. Thanks!

Reply:
I did that, and the tests fail! Something else is going on...

Reply:
nevermind - it's fine - it was my mistake, I'm deleting it now :)